aiohttp/web_fileresponse.py

```diff
@@ -94,12 +95,25 @@ def _done_fut(self, out_fd: int, fut: 'asyncio.Future[None]') -> None:
 
     async def sendfile(self) -> None:
         assert self.transport is not None
+        loop = self.loop
+        data = b''.join(self._sendfile_buffer)
```

Comment:
Would the use of `memoryview()` be faster in this case?

Reply:
I doubt that the speed difference is visible at all.
You can measure it if you want.